crypto/unstable_keystack.ts

```diff
@@ -145,7 +146,9 @@ export class KeyStack {
     return -1;
   }
 
-  [Symbol.for("Deno.customInspect")](inspect: (value: unknown) => string) {
+  [Symbol.for("Deno.customInspect")](
```

Comment:
How would we describe this?

Reply:
Maybe something like `/** Custom output for Deno.inspect */`